bot/exts/fun/latex.py

```diff
@@ -81,8 +88,11 @@ def __init__(self, bot: Bot):
     async def _generate_image(self, query: str, out_file: BinaryIO) -> None:
         """Make an API request and save the generated image to cache."""
         payload = {"code": query, "format": "png"}
-        async with self.bot.http_session.post(LATEX_API_URL, data=payload, raise_for_status=True) as response:
-            response_json = await response.json()
+        try:
```

Comment:
I'm wondering whether we should keep `raise_for_status` here since in this particular case, we lose access to the `response.json()` call, which contains the logs (per the docs)
![image](https://github.com/python-discord/sir-lancebot/assets/48383734/3ae80dec-4fbd-47d8-a7dc-5a6f6636f903)

But i don't think it would be necessary since it's a server-side error and we can't make much out of it, I was just thinking out loud.

Reply:
based on the server code here https://github.com/DXsmiley/rtex/blob/master/src/jobs.py#L38
I think all potential rendering issue are handled, therefore no logs will be provided in the case of 5xx responses; so keeping `raise_for_status` seems logical in this case

What do you think ?

Reply:
Yeah, i shouldn't have spoken out loud in the first place :P

The thing is, even if we have server logs, there's nothing we can do about it since we're only the client. (Unless you want to make a PR to the other repo to fix it :P)

So yes, let's keep it as is, since there's not much value to us here